### PR TITLE
Kcho/mediaflux test

### PR DESCRIPTION
**Pull request by kcho**
Mediaflux test working version.

Updates
- data types without `protected` specified to be protected by default
- keyring.__init__ update so the users without TOKEN can also pull the data
- `lochness_create_template` updated to pull the data structure shared to `Prescient`

### CODE REVIEW

**Comment by tashrifbillah** [comment]
Hi Kevin, what's the difference between this and PR #27 ?

Again, this title says mediaflux test but it has a whole bunch of updates in it. I would appreciate a phone call walkthrough :)

**Comment by kcho** [comment]
> Hi Kevin, what's the difference between this and PR #27 ?
> 
> Again, this title says mediaflux test but it has a whole bunch of updates in it. I would appreciate a phone call walkthrough :)

image

Since the mediaflux has been returning us error due to the token issue, I've separated the mediaflux testing branch out from the
PR#27. Please ignore everything above commit number 8173027, since they are included in the PR#27.

File: lochness/mediaflux/__init__.py [hunk 1]
```diff
@@ -127,17 +127,17 @@ def sync_module(Lochness: 'lochness.config',
                                 remote= remote.split(':')[1]
 
                             # construct local path
-                            protect = prod.get('protect', False)
+                            protect = prod.get('protect', True)
```
Comment (tashrifbillah) [inline lochness/mediaflux/__init__.py:130]: Are we putting data to `PROTECTED` as the default option?

Comment (tashrifbillah) [reply lochness/mediaflux/__init__.py:130]: Never mind, just saw the commit message :)

File: scripts/lochness_create_template.py [hunk 2]
```diff
@@ -233,26 +233,31 @@ def create_config_template(config_loc: Path, args: object) -> None:
...
-        namespace: /DATA/ROOT/UNDER/BOX
+        namespace: /DATA/ROOT/UNDER/MEDIAFLUX
         delete_on_success: False
         file_patterns:
             actigraphy:
                 - vendor: Philips
                   product: Actiwatch 2
-                  data_dir: all_BWH_actigraphy
-                  pattern: 'accel/*csv'
+                  data_dir: actigraphy
```
Comment (tashrifbillah) [inline scripts/lochness_create_template.py:242]: Just so we are on the same page--Justin showed the remote name to be more like the long name I have written: `all_BWH_actigraphy`. Your suggested change means a user would have to change the proposed `actigraphy` name to the actual remote name.

Comment (kcho) [reply scripts/lochness_create_template.py:242]: Yes, correct. This is the new structure we already shared to the Prescient and Pronet side, and they will try to match it to this example.

**Comment by tashrifbillah** [comment]
We got an interesting branch name --`kcho/mediaflux_text` ... only realized the last word after creating a local branch `kcho/mediaflux_test` and failing to `git pull` 🥇

**Comment by tashrifbillah** [comment]
`protected: False` is apparently not working. Even though I do that, things get pulled to `PROTECTED` directory!

**Comment by tashrifbillah** [comment]
Please ignore the false alarm, I wrote `protected` instead of the expected `protect`!

**Comment by tashrifbillah** [comment]
Why is the discrimintation for mediaflux?

```
[tb571@rc-predict lochness]$ grep -R ".get('protect'," *
box/__init__.py:                    protect = product.get('protect', False)
dropbox/mclean.py:                    protect = product.get('protect', False)
dropbox/nrg.py:                    protect = product.get('protect', False)
mediaflux/__init__.py:                            protect = prod.get('protect', True)
```

**Comment by tashrifbillah** [comment]
Tashrif has tested the following features:

1. BIDS: True/False
2. processed; True/False
3. protected: True/False
4. Skipping download of existing files

While Tashrif needs to update standalone `mediaflux.md` at a later time, this PR can be merged upon addressing the above comment.
@kcho

**Comment by kcho** [comment]
> Why is the discrimintation for mediaflux?
> 
> ```python
> [tb571@rc-predict lochness]$ grep -R ".get('protect'," *
> box/__init__.py:                    protect = product.get('protect', False)
> dropbox/mclean.py:                    protect = product.get('protect', False)
> dropbox/nrg.py:                    protect = product.get('protect', False)
> mediaflux/__init__.py:                            protect = prod.get('protect', True)
> ```

Thanks for pointing this out. I'll revisit this in `kcho/BIDS` branch.

**Pull request merged**